test/value_ref.cpp

```diff
@@ -92,6 +92,13 @@ class value_ref_test
             string const s{};
             (value_ref)s;
         }
+
+        // test that value_ref can be constructed
+        // from a const rvalue
```

Comment:
This comment can be terse:
```
// construct from const rvalue
```

The rest is implied by context.

Reply:
Alright, good to go.